js/index.js

```diff
@@ -0,0 +1,198 @@
+let playTurn = 1;
```

Comment:
we shouldn't make use of global code, let use modules or factories.
![image](https://user-images.githubusercontent.com/31897434/75405859-2b0ee780-590f-11ea-84e0-e2e66e5130ed.png)
https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript